Stripe/STPPaymentContext.m

```diff
@@ -406,16 +406,9 @@ - (void)requestPayment {
             return;
         }
         if (!self.selectedPaymentMethod) {
-            STPAddCardViewController *addCardViewController = [[STPAddCardViewController alloc] initWithConfiguration:self.configuration theme:self.theme];
-            addCardViewController.delegate = self;
-            addCardViewController.prefilledInformation = self.prefilledInformation;
-            addCardViewController.shippingAddress = self.shippingAddress;
-            UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:addCardViewController];
-            navigationController.navigationBar.stp_theme = self.theme;
-            navigationController.modalPresentationStyle = self.modalPresentationStyle;
-            [self.hostViewController presentViewController:navigationController animated:YES completion:nil];
+            [self presentPaymentMethodsViewController];
```

Comment:
do we need to update the payment methods VC delegate callback to call `requestPayment` again? (I don't think we were ever presenting payment methods VC from the `requestPayment` flow before)